libraries/Microsoft.Bot.Connector/OAuthApiClient.cs

```diff
@@ -0,0 +1,350 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Schema;
+using Microsoft.Rest;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Connector
+{
+    /// <summary>
+    /// Service client to handle requests to the botframework api service.
+    /// </summary>
+    public class OAuthClient : ServiceClient<OAuthClient>
+    {
+        private readonly ConnectorClient client;
```

Comment:
We use a leading "_" for member variables. So "_client" would be how to fix this.